cli.ts

```diff
@@ -43,4 +43,21 @@ program
     await new TestCommand(languageFilter).run();
   });
 
+program
+  .command("buildAndRun")
```

Comment:
```suggestion
  .command("build-and-run")
```

(Seems more common to name commands this way rather than without dashes/underscores)